Kopernicus/Kopernicus/RuntimeUtility/RuntimeUtility.cs

```diff
@@ -153,6 +153,13 @@ void Start()
                     body.orbitDriver.orbit = loader.orbit;
                     CelestialBody oldRef = body.referenceBody;
                     body.referenceBody.orbitingBodies.Remove(body);
+
+                    CelestialBody newRef = PSystemManager.Instance.localBodies.FirstOrDefault(b => b.transform.name == loader.referenceBody);
+                    if (newRef != null)
+                        body.orbit.referenceBody = body.orbitDriver.referenceBody = newRef;
+                    else
+                        throw new Exception("PostSpawnOrbit reference body for \"" + body.name + "\" could not be found. Missing body name is \"" + loader.referenceBody + "\".");
```

Comment:
Please make this a `Debug.Log`. The system loaded successfully, no need to kill the runtime maintainance when the system could work.

Reply:
That said, we should add a check that disables RuntimeUtility if the loader failed.

Reply:
not sure about "could work"

I mean, if it's designed to work with the PostSpawnOrbit and an user messes up when updating the planet pack to a newer version, it could result in a save loading on a system different than what it is supposed to be, potentially losing crafts and crybabies going nuts.

that said, I can change it if you are convinced Debub.Log will be a better choice

Reply:
Make it a Debug.Log and spawn the "something broke search help" window?